src/polynomials/primitives/bernstein_helpers.cpp

```diff
@@ -0,0 +1,42 @@
+#include "bernstein_helpers.hpp"
+#include "common.hpp"
+
+#include <numeric>
+
+namespace bebot::bernstein::internal
+{
+
+double binom(int n, int k)
+{
+    // Stolen from cppreference :P
+    // TODO: No idea how efficient this is
+    return 1 / ((n + 1) * std::beta(n - k + 1, k + 1));
```

Comment:
Clever, haven't seen this before. I did double check that it is mathematically valid, though I am not familiar with the std::beta function. We'll definitely want to check this function for speed in the future since the binomial function is called quite a few times. In the Python implementation, I tend to cache all of the calls since it is an insignificant amount of memory being used.

Reply:
I plan on adding memoization functionality later with something along the lines of Python's @lru_cache for other things. We can revisit performance stuff for binom after some profiling later

Reply:
That sounds great to me.